citi-onboarding/Aqua-Jr
# feature/navbar

## What I did
- Add the navbar component

## How to test
1. Access the branch through your terminal:
```
   git checkout develop
   git pull
   git checkout feature/navbar
```
2. Access the server folder and run the server:
```
   cd /server
   nodemon index.js
```

3. Access the client folder in another terminal window and start React:
```
   cd /client
   yarn start
```
4. In your browser, go to:
``` 
   localhost:3000
```
5. Check if the navbar is responsive (from 300px to 3000px)
6. Check if the side menu opens in the mobile version
7. Check if everything looks just like he mockup
